README.md

```diff
@@ -1 +1,128 @@
-# Decode-a-Morse-code-message
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230049928-8e3e8b64-6fa9-4623-80ff-0e54073e5177.png)

**[OPTIONAL]**

It would be very great if you make all your commit messages more meaningful and capitalize them because I saw your commit is not capitalized 😨 Kindly consider them making capitalized and meaningful to make your pull request professional 👍